contrib/ros/README.md

```diff
@@ -0,0 +1,138 @@
+##### intera.sh ROS Environment Setup
+##### Copy the intera.sh script
+The intera.sh file already exists in intera_sdk repo, copy the file into your ros workspace.
+    
+    $ cp ~/<YOUR_FAVORITE_WORKSPACE_NAME>/src/intera_sdk/intera.sh ~/<YOUR_FAVORITE_WORKSPACE_NAME>
+##### Customize the intera.sh script
+    $ cd ~/<YOUR_FAVORITE_WORKSPACE_NAME>
+    $ gedit intera.sh
+##### Edit the 'robot_hostname' field
+##### Edit the 'your_ip' field
+##### Verify 'ros_version' field
+update: ros_version='kinetic'
+##### Save and Close intera.sh script
+##### Initialize your SDK environment
+    $ cd ~/<YOUR_FAVORITE_WORKSPACE_NAME>
+    $ ./intera.sh
+### Setup Gazebo Simulation for Sawyer
+**Please skip this section if you don't want simulation**
+#### Installation/Prerequisites
+- Make sure you have finished [workstation setup](#setup-workstation)
+- Ensure the following software packages are installed:
+    
+    
+    $ sudo apt-get install gazebo7 ros-kinetic-qt-build ros-kinetic-gazebo-ros-control ros-kinetic-gazebo-ros-pkgs ros-kinetic-ros-control ros-kinetic-control-toolbox ros-kinetic-realtime-tools ros-kinetic-ros-controllers ros-kinetic-xacro python-wstool ros-kinetic-tf-conversions ros-kinetic-kdl-parser ros-kinetic-sns-ik-lib
+#### Sawyer Simulator Installation
+##### Install sawyer_simulator
+    $ cd ~/<YOUR_FAVORITE_WORKSPACE_NAME>/src
+    $ git clone https://github.com/RethinkRobotics/sawyer_simulator.git
+    $ wstool merge sawyer_simulator/sawyer_simulator.rosinstall
+    $ wstool update
+##### Build Source
+    $ source /opt/ros/kinetic/setup.bash
+    $ cd ~/<YOUR_FAVORITE_WORKSPACE_NAME>
+    $ catkin_make -DPYTHON_EXECUTABLE:FILEPATH=<path_to_your_rllab_env_python_interp>
+### Convert Python2 code in intera packages to Python3
+- As our environments only support Python3, we need to convert Python2 code in intera packages.
+- These can be done using rllab.contrib.ros.scripts.sawyer_2to3.sh
+##### Customize sawyer_2to3.sh
+Upgrade path:
+    
+    ros_ws=<YOUR_FAVORITE_WORKSPACE_PATH>/src/
+Remove sawyer_simulator, if you don't need it:
+
+    packages=(intera_sdk)
+##### Execute script
+## Usage
+### Start the gazebo simulation
+- This section is only for simulation user
+#### Copy the sawyer simulation launch file and simulation world file
+    $ cp <rllab.contrib.ros.envs.sawyer.sawyer_learning.launch, rllab.contrib.ros.envs.sawyer.sawyer_world_learning.launch> <YOUR_FAVORITE_WORKSPACE_PATH>/src/sawyer_simulator/sawyer_gazebo/launch
+    $ cp <rllab.contrib.ros.envs.sawyer.sawyer_learning.world> <YOUR_FAVORITE_WORKSPACE_PATH>/src/sawyer_simulator/sawyer_gazebo/worlds/
+#### Set the environment variables
+    $ cd ~/<YOUR_FAVORITE_WORKSPACE_NAME>
+    $ ./intera.sh sim
+    $ export PYTHONPATH=/opt/ros/kinetic/lib/python2.7/dist-packages:<YOUR_FAVORITE_WORKSPACE_PATH>/devel/lib/python3/dist-packages
+#### Launch the gazebo sawyer
+    $ roslaunch sawyer_gazebo sawyer_learning.launch
```

Comment:
apt-install python-rospkg

Reply:
pip install rospkg

Reply:
pip install catkin_pkg

Reply:
No module named defusedxml.xmlrpc
pip install defusedxml

Reply:
Make sure to tell the users that the very first time launching GAZEBO the program will take a long time to open, but it's normal because it's downloading extra packages

Reply:
@ryanjulian Where should @hjzh4 list the dependencies? At the end of the README?

Reply:
Inline as apt/pip installation instructions, plus a brief bullet list of major deps at the top section where he mentions the system configuration this assumes.

Reply:
Listed the python packages dependencies like
## Support Platform
| Robot        | Workstation OS | ROS    | Python | Simulation Platform | 
|:------------:|:--------------:| :-----:|:------:|:-------------------:|
| Sawyer       | Ubuntu16.04    | Kinetic| 3      |Gazebo 7.0           |
### Major python packages dependencies
- catkin_pkg
- rospkg
- defusedxml
Added the installation intructions
##### Install Python Dependecies
```bash
$ source activate rllab3
$ pip install catkin_pkg
$ pip install rospkg
$ pip install defusedxml
```